.pre-commit-config.yaml

```diff
@@ -31,7 +31,7 @@ repos:
     hooks:
       - id: codespell
         args:
-          - --ignore-words-list=additionals,alle,alot,ba,bre,bund,datas,dof,dur,ether,farenheit,falsy,fo,haa,hass,hist,iam,iff,iif,incomfort,ines,ist,lightsensor,mut,nam,nd,pres,pullrequests,referer,resset,rime,ser,serie,sur,te,technik,ue,uint,unsecure,visability,wan,wanna,withing,zar
+          - --ignore-words-list=additionals,alle,alot,ba,bre,bund,datas,dof,dur,ether,farenheit,falsy,fo,haa,hass,hist,iam,iff,iif,incomfort,ines,ist,lightsensor,mut,nam,nd,pres,pullrequests,referer,resset,rime,ser,serie,sur,te,technik,ue,uint,unsecure,visability,wan,wanna,withing,zar,currenty
```

Comment:
Please sort 🔡.

Reply:
That's related to this `https://github.com/home-assistant/core/blob/679642e6d2c8fc352184060db9b1253c375c1d14/homeassistant/components/matter/light.py#L199` codespell wants to rename it to "currently", don't know if there's another way of fixing it.